### DISCUSSION TITLE
高精度模型哪里下载？

### DISCUSSION THREAD

**Discussion by playgithub** [PaddlePaddle/PaddleOCR#13444] [Q&A] [answered]
很多简单的识别不到
![image](https://github.com/user-attachments/assets/523f1b3c-d110-4855-bd52-bff3d78838ef)

**Comment by GreatV** [comment, marked as answer]
https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/models_list.md

**Reply by playgithub** [reply]
![image](https://github.com/user-attachments/assets/63bfe7ab-ed55-4c1c-afd0-912ce11c1c01)

Much better now, but still some obvious characters can't be detected, like "1" and "Y", any suggestion?

Thanks

**Reply by playgithub** [reply]
For a gray image, it's almost perfect, except that two single "1"s can't be dectected.
![image](https://github.com/user-attachments/assets/dc1562a1-bb85-4c2c-bc6b-a07a8fca3419)

I think "1" and "l" and "I" could be abiguous,
so a model for only numbers should be a workaround,
do ocr two times, one for characters and one for numbers,
and an algorithm that merges the results could be simple for a regular image.

Are there number-only models?
Or any better solution?
